app/views/dashboard/advisers/_advisers_table.html.erb

```diff
@@ -9,18 +9,29 @@
       </tr>
     </thead>
     <tbody>
-      <% advisers.each do |adviser| %>
-        <tr class="t-advisers-table-row">
-          <td class="t-adviser-reference-number"><%= adviser.reference_number %></td>
-          <td class="t-adviser-name"><%= adviser.name %></td>
-          <td>
-            <ul class="l-comma-separated-list">
-              <% adviser.qualifications.each do |qualification| %>
-                <li class="t-adviser-qualification"><%= qualification.name %></li>
-              <% end %>
-            </ul>
+      <% if firm.advisers.present? %>
+        <% firm.advisers.each do |adviser| %>
+          <tr class="t-advisers-table-row">
+            <td class="t-adviser-reference-number"><%= adviser.reference_number %></td>
+            <td class="t-adviser-name"><%= adviser.name %></td>
+            <td>
+              <ul class="l-comma-separated-list">
```

Comment:
Yeah, its difficult to explain. So a component is just a section of the UI that could be re-used elsewhere. It'll have a 'default' style, but can be modified. In this case, its just a list. The list could be displayed horizontally or vertically, using a modifier class.

This list can then be positioned on the page using a layout, which will typically set up the grid. You'd want to avoid nesting layouts within layouts too much as that's probably a symptom of the components not being styled flexibly enough in the first place. The layout can override specific styles in the component, but too much of this probably indicates the component itself needs working on.

I guess you could think that if you're using a layout just to style an `li` type element, then that's too low level.

Remember, overall, this is a continuously evolving practice and is HARD. Devices, best practices, features are constantly changing which is why we can't give you 'how to use the microwave' style instructions. Hopefully this helps a little bit.